service/project.go

```diff
@@ -42,6 +42,15 @@ func (uis *UIServer) filterViewableProjects(u *user.DBUser) ([]model.ProjectRef,
 func (uis *UIServer) projectsPage(w http.ResponseWriter, r *http.Request) {
 	dbUser := MustHaveUser(r)
 
+	flags, err := evergreen.GetServiceFlags()
+	if err != nil {
+		gimlet.WriteResponse(w, gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "retrieving admin settings")))
+	}
+	if flags.LegacyUIProjectPageDisabled {
+		newUIProjectsLink := fmt.Sprintf("%s/projects", uis.Settings.Ui.UIv2Url)
+		http.Redirect(w, r, newUIProjectsLink, http.StatusTemporaryRedirect)
```

Comment:
should this be `StatusPermanentRedirect` since we are planning to fully deprecate it?

Reply:
I said temporary in that there will be a temporary redirect after which the page won't exist at all.  That being said I can change it if you prefer

Reply:
That's a good point actually. I'm fine with either, I'll leave it up to you.